### ISSUE TITLE
Add analytics

### ISSUE THREAD

**Issue by RyanWarner** [GatsbyGoodies/gatsby-goodies-site#13]
We should have analytics to track unique visits, device metrics, etc.

I like amplitude a lot

**Comment by ssw1ss** [comment]
I agree this is important. I have no experience with analytics tools outside of google analytics, but we can definitely use Amplitude if you like.
